app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -0,0 +1,47 @@
+import { TextInput, Button } from '@18f/identity-components';
+import { request } from '@18f/identity-request';
+import { useState, useCallback } from 'react';
+
+interface Location {
+  street_address: string;
+  city: string;
+  state: string;
+  zip_code: string;
```

Comment:
just wondering if it matters at all if a user inputs the longer form of a zip code (not that it's very likely)?